api.go

```diff
@@ -321,6 +356,13 @@ func (c *Client) Authenticate(username, password, computeid, vdcid string) (Vdc,
 // set.
 func (c *Client) NewRequest(params map[string]string, method string, u url.URL, body io.Reader) *http.Request {
 
+	debug := os.Getenv("VCLOUDAIR_SHOW_BODY")
```

Comment:
The implementation is disruptive currently since it empties the buffer before the raw bytes can be parsed.  I brought it into another setting to allow more flexibility when debugging.